### DISCUSSION TITLE
RTL direction in theme

### DISCUSSION THREAD

**Discussion by mahmoud-m-abadi** [filamentphp/filament#371] [Help]
Hi,
Thanks for your great work.
I need RTL support in the theme. Can I do anything to fix this?
Please give me some tricks.

Thanks in advance.

**Comment by danharrin** [comment]
Hey, a PR is open for this - #367 - but I think some of the UI needs tweaking. Feel free to suggest improvements on that thread!

**Reply by mahmoud-m-abadi** [reply]
Thanks for suggest this, it works great!
Please add it in next version!!
